### PR TITLE
Bump plugin version and dependency upgrades

### PR DESCRIPTION
**Pull request by melix**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -20,8 +20,16 @@ jobs:
     - name: Set up JDK
       uses: actions/setup-java@v2
       with:
-        distribution: 'adopt'
+        distribution: 'temurin'
         java-version: '8'
+    - name: Check plugin
+      run: ./gradlew check
+    - name: Publish Test Report
+      if: always()
```
Comment (alvarosanchez) [inline .github/workflows/gradle.yml:28]: This is redundant, isn't it?

**Review by alvarosanchez — APPROVED** [review]

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (melix) [reply .github/workflows/gradle.yml:28]: Why? Before this change tests weren't executed on CI.

Comment (alvarosanchez) [reply .github/workflows/gradle.yml:28]: My bad, I thought `always()` was the default, but in reality it is `success()`.

BTW, according to [the docs](https://docs.github.com/en/actions/learn-github-actions/expressions#always), it seems it should be:

```suggestion
      if: ${{ always() }}
```

Comment (melix) [reply .github/workflows/gradle.yml]: Interesting, we use that notation in several places I think

**Pull request merged**